src/assets/locales/en.js

```diff
@@ -146,6 +146,7 @@ export default {
         valid: "",
         invalid: "Mobile number is invalid",
       },
+      input_placeholder: "Mobile Number",
```

Comment:
placeholder should also not be in the center - it should be in the left - like the way we have in Editor